script.module.nexx/addon.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="script.module.nexx" 
+	     name="Nexx Video API service module" 
+	     version="1.2.0" 
+	     provider-name="sarbes">
+  <requires>
+    <import addon="xbmc.python" version="2.25.0" />
+    <import addon="script.module.libmediathek3" version="1.5.0"/>
+  </requires>
```

Comment:
Thanks for the update, I noticed you are missing a assets parameter. If you could add this to your next update I'd appreciate it. THX
https://kodi.wiki/view/Addon.xml#.3Cassets.3E